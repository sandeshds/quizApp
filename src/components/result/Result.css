td {
    border-bottom: 1px solid #ddd;
    padding: 15px;
    text-align: left;
}

table {
    width:fit-content;
    height:fit-content;
    position:fixed;
    margin-left:-150px; /* half of width */
    margin-top:-150px;  /* half of height */
    top:50%;
    left:50%;
}

h2 {
    text-align: center;
    font-size: xx-large;
}

.resultsBlock {
    font-size: larger;
    background-color: #0E5D47;
    color: white;
    height: -webkit-fill-available;
    border: 20px solid #6E320D;
}

.buttonRetake {
    background: transparent;
    border: 1px solid white;
    color: white;
    height: fit-content;
    width: fit-content;
    font-size: large;
    position: absolute;
    left: 50%;
    margin-left: -50px;
    bottom:10%;
}