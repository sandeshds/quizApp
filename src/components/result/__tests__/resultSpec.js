import React, {Component} from 'react';
import renderer from 'react-test-renderer';
import Result from './../index';

describe('Button', ()=>{
    it('should render result properly', ()=>{
        let results = {
            totalNumberOfQuestions : 8,
            questionsAnsweredCorrectly : 2,
            percentageScored : 25
        };
        let tree = renderer.create(
            <Result 
                results = {results}
            />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});