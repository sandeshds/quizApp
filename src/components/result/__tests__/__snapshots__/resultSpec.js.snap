// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Result should render result properly 1`] = `
<div
  className="resultsBlock"
>
  <h2>
     Results 
  </h2>
  <table>
    <tr>
      <td>
        Total No of questions
      </td>
      <td>
        8
      </td>
    </tr>
    <tr>
      <td>
        No of questions answered correctly
      </td>
      <td>
        2
      </td>
    </tr>
    <tr>
      <td>
        Percentage Scored
      </td>
      <td>
        25
         %
      </td>
    </tr>
  </table>
  <input
    className="buttonRetake"
    onClick={[Function]}
    type="button"
    value="Retake Quiz"
  />
</div>
`;
